packages/insomnia-app/app/ui/components/wrapper-unit-test.js

```diff
@@ -0,0 +1,472 @@
+import Editable from './base/editable';
+import type { SidebarChildObjects } from './sidebar/sidebar-children';
+import SelectModal from './modals/select-modal';
+import type { UnitTestSuite } from '../../models/unit-test-suite';
+import ActivityToggle from './activity-toggle';
+import { getSendRequestCallback } from '../../common/send-request';
+import type { GlobalActivity } from '../../common/constants';
+import { ACTIVITY_HOME } from '../../common/constants';
+
+type Props = {|
+  children: SidebarChildObjects,
+  gitSyncDropdown: React.Node,
+  handleActivityChange: (workspaceId: string, activity: GlobalActivity) => Promise<void>,
+  wrapperProps: WrapperProps,
+|};
+
+type State = {|
+  testsRunning: Array<UnitTest> | null,
+|};
+
+@autobind
+class WrapperUnitTest extends React.PureComponent<Props, State> {
+  state = {
+    testsRunning: null,
+  };
+
+  // Defining it here instead of in render() so it won't act as a changed prop
+  // when being passed to <CodeEditor> again
+  static lintOptions = {
+    globals: {
+      // https://jshint.com/docs/options/
+      insomnia: true,
+      expect: true,
+      chai: true,
+      debugger: true,
+    },
+    asi: true, // Don't require semicolons
+    undef: true, // Prevent undefined usages
+    node: true, // Enable NodeJS globals
+    esversion: 8, // ES8 syntax (async/await, etc)
+  };
+
+  autocompleteSnippets(unitTest: UnitTest): Array<{ name: string, value: () => Promise<string> }> {
+    const sendReqSnippet = (requestId: string) => {
+      for (let i = 1; i < 100; i++) {
+        const variableName = `response${i}`;
+        if (!unitTest.code.includes(`const ${variableName} =`)) {
+          return (
+            `const ${variableName} = await insomnia.send(${requestId});\n` +
+            `expect(${variableName}.status).to.equal(200);`
+          );
+        }
+      }
+    };
+
+    return [
+      {
+        name: 'Send Current Request',
+        displayValue: '',
+        value: sendReqSnippet(''),
+      },
+      {
+        name: 'Send Request By ID',
+        displayValue: '',
+        value: async () => {
+          return new Promise(resolve => {
+            showModal(SelectModal, {
+              title: 'Select Request',
+              message: 'Select a request to fill',
+              value: '__NULL__',
+              options: [
+                { name: '-- Select Request --', value: '__NULL__' },
+                ...this.buildSelectableRequests().map(({ name, request }) => ({
+                  name: name,
+                  displayValue: '',
+                  value: sendReqSnippet(`'${request._id}'`),
+                })),
+              ],
+              onDone: v => resolve(v),
+            });
+          });
+        },
+      },
+    ];
+  }
+
+  async _handleCreateTestSuite() {
+    const { activeWorkspace } = this.props.wrapperProps;
+    showPrompt({
+      title: 'New Test Suite',
+      defaultValue: activeWorkspace.name + ' Suite',
```

Comment:
```suggestion
      defaultValue: activeApiSpec.fileName + ' Suite',
```

Currently when you rename an api spec it does not rename the workspace, meaning this suit name prompt wouldn't make sense. I suggest we set the default value to simply be "New Suite", or use `apiSpec.fileName`.

Reply:
I like "New Suite"